### PR TITLE
feat: Hapi.js starter template

### PR DESCRIPTION
**Pull request by TheSTL**
**What kind of change does this PR introduce?**
Feature: Support `Hapi.js` Nodejs framework (Except for Graphql template).

**Did you add tests for your changes?**
Yes.
**If relevant, did you update the documentation?**
No
**Summary**

Support Hapi.js Nodejs framework (except for Graphql template).

Addresses #66 

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by TheSTL** [comment]
@jamesgeorge007 You can review this PR.

File: src/commands/basic/init.js [hunk 1]
```diff
@@ -156,17 +156,40 @@ const fetchTemplate = async (template) => {
...
+  // Doesn't support more option for GraphQl
+  if (template !== 'GraphQL') {
+    requireServerChoices.push('More');
+  }
+  // Show up a suitable prompt whether if the user requires a Full stack application (Default: Express.js)
+  const { requireServerOption } = await inquirer.prompt({
+    type: 'list',
     message: 'Do you require server side template (Express.js)',
+    name: 'requireServerOption',
+    choices: requireServerChoices,
```
Comment (jamesgeorge007) [inline src/commands/basic/init.js]: Let's keep the `confirm` prompt as it is, and if the user prefers to have a server-side template, ask his/her framework of choice (`Express.js`/`Hapi.js`).

Comment (TheSTL) [reply src/commands/basic/init.js]: I think MEVN will lose it's meaning if the user prefers the server-side template and then again he has to choose a server-side template from options. That's why I made it as a list. If a user prefers a server-side template then by default it should be `express.js` @jamesgeorge007

Comment (jamesgeorge007) [reply src/commands/basic/init.js]: We shouldn't have supported any of those frameworks if that was the case. Please make the suggested change, and we are good to go.

Comment (TheSTL) [reply src/commands/basic/init.js]: Sure :+1:

**Comment by TheSTL** [comment]
@jamesgeorge007 Please review again.

**Review by jamesgeorge007 — CHANGES_REQUESTED** [review]
Looks good; can you also address the case when the user needs to generate a `CRUD boilerplate` with the [generate](https://github.com/madlabsinc/mevn-cli/blob/develop/src/commands/basic/generate.js) command?

**Comment by TheSTL** [comment]
Done with `mevn-generate` command changes for hapi @jamesgeorge007

**Review by jamesgeorge007 — APPROVED** [review]

**Pull request merged**